scripts/common/get_engine_management.sh

```diff
@@ -17,20 +17,20 @@
 ##--------------------------------------------------------------------
 
 ##
-## write_log
 ## This script is used to get information regarding the Storage Plugin
-## from the FogLAMP configuration file $FOGLAMP_DATA/etc/foglamp.json
 
-get_engine_management() {
 
-    # Remove new lines and store in a variable
-    json_foglamp=`tr -d '\n' < $FOGLAMP_DATA/etc/foglamp.json`
+get_engine_management() {
 
-    # Remove tabs
-    json_foglamp=`echo $json_foglamp | tr -d '\t'`
+  storage_info=`$FOGLAMP_ROOT/scripts/services/storage --plugin`
```

Comment:
Easier way to do this is to make storage_info an array
storage_info=(`$FOGLAMP_ROOT/scripts/services/storage --plugin`)
storage=${storage_info[0]}
managed=${storage_info[1]}

saves running echo and cut twice each

Reply:
that would be:

storage_info=( $($FOGLAMP_ROOT/scripts/services/storage --plugin) )